tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1286,8 +1286,13 @@ public function updateSpaceName(
 		string $spaceName,
 		string $newName
 	): void {
-		$space = $this->getSpaceByName($user, $spaceName);
-		$spaceId = $space["id"];
+		if ($spaceName === "unknown") {
+			// check sending invalid data
+			$spaceId = "39c49dd3-1f24-4687-97d1-42df43f71713";
```

Comment:
we have a helper function to generate the UUID

https://github.com/owncloud/core/blob/a890932216b9450d7cc7446dda7163fc8f999eab/tests/TestHelpers/WebDavHelper.php#L427-L436
```suggestion
			$spaceId = WebDavHelper::generateUUIDv4();
```